docs/docker.md

```diff
@@ -1,18 +1,46 @@
-# Docker Install
+# Docker
 
 These instructions explain how to run Feldera on a single machine
 in a configuration suitable for demos, development, and testing.  For production
-use, or for developing parts of Feldera itself, Feldera
-supports other forms of deployment.
+use, check out [Feldera Enterprise](/enterprise).
 
-## Install prerequisites
+## Docker Quickstart
 
-To run the demo, you will first need to install Docker and the Docker
-Compose plugin.  If you don't already have them, install them one of
-these ways:
+```
```

Comment:
Do we have a way to auto-update these docs at release time? Shouldn't this be `:latest`?

Reply:
I think latest == nightly builds. I need to update the release script to auto-update that.